github/lib/helperMessage.ts

```diff
@@ -29,9 +29,11 @@ export async function helperMessage({
     5) Get Recent Pull Request of a Repository -> /github  Username/RepositoryName pulls
     6) Review a Pull Request -> /github  Username/RepositoryName pulls pullNumber
     7) Login to GitHub -> /github login
-    8) View/Add/Delete/Update Repository Subscriptions -> /github subscribe 
-    9) Subscribe to all repository events -> /github Username/RepositoryName subscribe
-    10) Unsubscribe to all repository events -> /github Username/RepositoryName unsubscribe
+    8) Logout from GitHub -> /github logout
```

Comment:
Instead of having 7 and 8.   Is it possible to have a better user experience of ....

a) logging in within the app settings session
b) from time to time, the app will ask the user to re-enter credential for security

The will gave a better "set and forget" experience to most users.   Casual users will never be bothered with logging in and logging out.

Reply:
@Sing-Li We cannot have the login phase in the apps settings session because the app settings are only visible to the users which have `Application Access` given by the `Server Admin` so I don't think it will be convenient.
The logout feature will not be used by most users, its just there for convenience incase anyone wants to remove credentials from the server. The Login will be done by the user through the slash command, but they will only be prompted to login while using some feature which requires auth. Apart from that, using `RC Scheduler` we will automatically logout the user periodically and they will be sent a direct message by `GitHub App bot` to login again. It follows the similar workflow as you have mentioned in `b)`, the user does not even have to enter credentials, they will be logged in directly once the click on the `login` button as the token is regenerated by GitHub. This was suggested by @RonLek  to keep the system scalable and not store tokens forever in App Memory. I have added the reasoning behind it in [Wiki/Auth](https://github.com/RocketChat/Apps.Github22/wiki/Authentication-using-RC-OAuth2-Module-and-RC-Apps-Scheduler#logout-user-using-scheduler-api).
Let me know if I am missing something, maybe we can improve the login user experience in some way, we can discuss this in the upcoming the weekly meeting :sweat_smile: